src/shared/containers/SetOrg.tsx

```diff
@@ -313,7 +313,7 @@ const SetOrg: FC = () => {
           <Route
             exact
             path={`${orgPath}/${SETTINGS}`}
-            component={VariablesIndex}
+            component={isNewIOxOrg ? LabelsIndex : VariablesIndex}
```

Comment:
Instead of conditionally rendering variables or labels, (which will persist after the conditions that require the conditional are gone) why don't we just make Labels the default page under settings? That way it'll just work, whether the org has variables or not

Reply:
We could do that - it'll just mean it changes the UI experience for users who aren't new IOx.

@garylfowler @gunnaraasen Does it matter whether the "Labels" page or the "Variables" page renders by default when users visit Settings? If we switch from Variables to "Labels" we won't need a different routing behavior for the "Settings" page depending on whether the user is in a New IOx org or not.

Reply:
I don't think it matters.

Reply:
Unless they disagree, I'm inclined to leave it as-is for now, just because Labels isn't the first tab. Reordering the tabs out of alpha order seems like a little yak-shave-y for just fixing the default page.